### ISSUE TITLE
Tests for twisted.manhole should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7719]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7719|
|Type|enhancement|
|Created|2014-11-23 11:01:42Z|
|Branch|https://github.com/twisted/twisted/tree/manhole-tests-names-7719|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7719.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/797/797c8c58ce06de5f52949ad1d8ba2cecad5d1562/2513aa63f9991728a36c5408b427a6aed80fe568.patch) (853 bytes) - added by eeshangarg on 2014-11-23 11:11:17Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7719 7719
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_manhole_tests_names_7719 branches/manhole-tests-names-7719
branch_author__eeshangarg eeshangarg
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1416740502117833 1416740502117833
changetime__1417032842572957 1417032842572957
version__None None
owner__exarkun exarkun
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg removed owner|
|-|-|

Hi!

I just added a patch(bugfix_7719.patch​) that changes all of the tests in twisted.manhole to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks for letting me work on this!

Regards,
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [43510]) Branching to 'manhole-tests-names-7719'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

(In [43511]) Apply bugfix_7719.patch 

Refs [#7719](https://github.com/twisted/twisted/issues/7719)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

Thank.  These changes look fine to me and Buildbot is pretty green.  I'll merge this.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [43514]) Merge manhole-tests-names-7719

Author: eeshangarg
Reviewer: exarkun
Fixes: [#7719](https://github.com/twisted/twisted/issues/7719)

Rename all of the test case classes in `twisted.manhole` to conform to the
recently documented test case class naming convention.
